gui/CMakeLists.txt

```diff
@@ -96,9 +96,10 @@ if(USE_BREAKPAD)
 endif()
 
 if (WIN32)
-	add_executable(${PROJECT_NAME} WIN32 ${SOURCES} ${FORMS} ${UTILS_SOURCES})
+	add_executable("Grabber" WIN32 ${SOURCES} ${FORMS} ${UTILS_SOURCES})
 else()
-	add_executable(${PROJECT_NAME} ${SOURCES} ${FORMS} ${UTILS_SOURCES})
+	add_executable("Grabber" ${SOURCES} ${FORMS} ${UTILS_SOURCES})
```

Comment:
That will break all CI systems.
Can't this renaming be done with the `RENAME` argument of install() instead?